assets/stylesheets/home.css

```diff
@@ -0,0 +1,198 @@
+html {
+  background-color: gray;
+  background-image: url("../images/hero-small.jpg");
+  background-size: cover;
+  background-blend-mode: multiply;
+}
+
+.flex-container{
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  .column-flow {
+    -webkit-flex-flow: column nowrap;
+    -ms-flex-flow: column nowrap;
+    flex-flow: colum nowrap;
+  }
+  .row-flow {
+    -webkit-flex-flow: row wrap;
+    -ms-flex-flow: row wrap;
+    flex-flow: row wrap;
+  }
+}
+
+header {
+  background-image: url("../images/hero-small.jpg");
+  background-size: cover;
+  width: 100%;
+}
+
+h1 span {
+  color: white;
+  font-size: 1rem;
+}
+body {
+  min-width: 220px;
+  margin: auto;
+}
+
+nav {
+  ul li {
+    list-style: none;
+  }
+  a {
+    display: inline-block;
+    font-weight: 400;
+  }
+  img {
+    width: 100%;
+  }
+  .home {
+    margin-left: 1.3em;
```

Comment:
Would strongly prefer `rem`s to `em`s, but that could be a future change. https://j.eremy.net/confused-about-rem-and-em/